java/configuration/src/main/java/sleeper/configuration/properties/instance/GarbageCollectionProperty.java

```diff
@@ -27,11 +27,17 @@ public interface GarbageCollectionProperty {
             .defaultValue("15")
             .propertyGroup(InstancePropertyGroup.GARBAGE_COLLECTOR)
             .runCdkDeployWhenChanged(true).build();
+    UserDefinedInstanceProperty GARBAGE_COLLECTOR_LAMBDA_TIMEOUT_IN_MINUTES = Index.propertyBuilder("sleeper.gc.lambda.timeout")
```

Comment:
The property name could include the units, like `sleeper.gc.lambda.timeout.minutes`.